iothub/device/src/ModuleClient.cs

```diff
@@ -834,6 +834,21 @@ private async Task<MethodResponse> InvokeMethodAsync(Uri uri, MethodRequest meth
 
             try
             {
+                var context = new PipelineContext
+                {
+                    ProductInfo = new ProductInfo
+                    {
+                        Extra = InternalClient.ProductInfo
+                    }
+                };
+
+                var transportSettings = new Http1TransportSettings();
```

Comment:
Why does this unconditionally create a `Http1TransportSettings` without checking if there is one available from options?

Reply:
Module clients can only use AMQP/MQTT for most operations. This one operation uses HTTP, though. This is kind of similar to how we have a separate FileUploadTransportSettings so that AMQP device clients can still configure file upload HTTP operations. Let's discuss this a bit more in this thread with Abhipsa:

https://github.com/Azure/azure-iot-sdk-csharp/pull/3293#discussion_r1174091015

Reply:
IOW connection lease timeout is not a concert on edge module

Reply:
I left a comment to that effect for future reference